src/app/components/login/login.js

```diff
@@ -57,7 +57,9 @@ app.component('login', {
           passphrase: this.Passphrase.normalize(_passphrase),
           network: this.network,
         });
-        this.$state.go(this.$rootScope.landingUrl || 'main.transactions');
+        this.$state.go(this.$rootScope.landingUrl && this.$rootScope.landingUrl !== 'main.forging' ?
```

Comment:
Please remove this part and make sure the message in `src/app/components/forging/forging.pug` will be shown if the user is not delegate or the registration is not confirmed,